.github/workflows/docs.yml

```diff
@@ -98,3 +98,21 @@ jobs:
           path: /tmp/pr_number.txt
           retention-days: 1
           if-no-files-found: error
+
+  r:
+    if: github.event_name != 'pull_request' || github.event.pull_request.draft == false
```

Comment:
When do we want this job to run? Could it be `github.event_name == 'pull_request'`?

Reply:
- If pull_request, it should run if the PR is not a draft (= ready for review)
- If not pull_request, it should always run.